src/js/components/Main/Home/Home.module.css

```diff
@@ -4,19 +4,167 @@
 
 .tipUl {
   text-align: left;
-
 }
 
 .tip {
+  z-index: 1;
+  background-color: white;
+  position: absolute;
+  width: 100%;
 
+  margin-top: 1rem;
   text-align: center;
 }
 
-.tipLi
-{
+.tipLi {
+  padding-left: 2rem;
+  font-size: 1.5rem;
+  padding-top: .2rem;
+  padding-bottom: .4rem;
+}
+
+.tipLi:hover {
+  /*background-color: #cecece;*/
+  color: white !important;
+  background: linear-gradient(90deg, rgba(0, 0, 0, 1) 0%, rgba(255, 255, 255, 1) 0%, rgba(255, 255, 255, 1) 100%);
+  animation: leftToRight .1s linear forwards;
+
+}
+
+@keyframes leftToRight {
+  0% {
+    background: linear-gradient(90deg, rgba(0, 0, 0, 1) 0%, rgba(255, 255, 255, 1) 0%, rgba(255, 255, 255, 1) 100%);
+  }
+  10% {
+    background: linear-gradient(90deg, rgba(0, 0, 0, 1) 10%, rgba(255, 255, 255, 1) 10%, rgba(255, 255, 255, 1) 100%);
+  }
+  20% {
+    background: linear-gradient(90deg, rgba(0, 0, 0, 1) 20%, rgba(255, 255, 255, 1) 20%, rgba(255, 255, 255, 1) 100%);
+  }
+  30% {
+    background: linear-gradient(90deg, rgba(0, 0, 0, 1) 30%, rgba(255, 255, 255, 1) 30%, rgba(255, 255, 255, 1) 100%);
+  }
+  40% {
+    background: linear-gradient(90deg, rgba(0, 0, 0, 1) 40%, rgba(255, 255, 255, 1) 40%, rgba(255, 255, 255, 1) 100%);
+  }
+  50% {
+    background: linear-gradient(90deg, rgba(0, 0, 0, 1) 50%, rgba(255, 255, 255, 1) 50%, rgba(255, 255, 255, 1) 100%);
+  }
+  60% {
+    background: linear-gradient(90deg, rgba(0, 0, 0, 1) 60%, rgba(255, 255, 255, 1) 60%, rgba(255, 255, 255, 1) 100%);
+  }
+  70% {
+    background: linear-gradient(90deg, rgba(0, 0, 0, 1) 70%, rgba(255, 255, 255, 1) 70%, rgba(255, 255, 255, 1) 100%);
+  }
+  80% {
+    background: linear-gradient(90deg, rgba(0, 0, 0, 1) 80%, rgba(255, 255, 255, 1) 80%, rgba(255, 255, 255, 1) 100%);
+  }
+  90% {
+    background: linear-gradient(90deg, rgba(0, 0, 0, 1) 90%, rgba(255, 255, 255, 1) 90%, rgba(255, 255, 255, 1) 100%);
+  }
+  100% {
+    color: white;
+    background: linear-gradient(90deg, rgba(0, 0, 0, 1) 100%, rgba(255, 255, 255, 1) 100%, rgba(255, 255, 255, 1) 100%);
+
+  }
+}
```

Comment:
How to re-write this animation in better way?

Reply:
to be honest is okay like you have. of course you can read about best practices in keyframes, [for example](https://blog.gyrosco.pe/smooth-css-animations-7d8ffc2c1d29) since you use keyframes in your code.